File: src/main/java/team4/footwithme/stadium/util/SortFieldMapper.java
package team4.footwithme.stadium.util;

import java.util.Map;

public class SortFieldMapper {
    private static final Map<String, String> SORT_FIELD_MAP = Map.of(
            "COURT", "courtId",
            "STADIUM", "stadiumId",
            "NAME", "name",
            "ADDRESS", "address"
    );

    public static String getDatabaseField(String sortField) {
        return SORT_FIELD_MAP.getOrDefault(sortField, "name");
    }
}

File: src/main/java/team4/footwithme/stadium/util/SortFieldMapper.kt
package team4.footwithme.stadium.util

object SortFieldMapper {
    private val SORT_FIELD_MAP: Map<String?, String> = java.util.Map.of(
        "COURT", "courtId",
        "STADIUM", "stadiumId",
        "NAME", "name",
        "ADDRESS", "address"
    )

    fun getDatabaseField(sortField: String?): String {
        return SORT_FIELD_MAP.getOrDefault(sortField, "name")
    }
}